s3.tf

```diff
@@ -0,0 +1,38 @@
+terraform {
+  backend "s3" {
+    bucket = "eks.bucket"
+    key    = "project/terraform.tfstate"
+    region = "us-east-1"
+  }
+}
+
+module "s3_bucket" {
+  source = "terraform-aws-modules/s3-bucket/aws"
+
+  bucket = "eks.bucket"
+  acl    = "public-read-write"
```

Comment:
this setting makes all resources in your bucket public, please disable it

Reply:
updated acl value to private